google-cloud-clients/google-cloud-logging/src/test/java/com/google/cloud/logging/LoggingImplTest.java

```diff
@@ -83,8 +83,11 @@
   private static final String PROJECT = "project";
   private static final String PROJECT_PB = "projects/" + PROJECT;
   private static final String SINK_NAME = "sink";
+  private static final SinkInfo.VersionFormat VERSION = SinkInfo.VersionFormat.V2;
```

Comment:
Why do we need a static instance of a static variable? 
Can we not directly use `V2`? Maybe do static import and make it `VersionFormat.V2`

Reply:
for this case we can directly use v2 but its better to use static global so object will be shared with other test cases

Reply:
What exactly do you mean by "static global"? I'd take it as a static member variable.

I believe using a static import saves you a reference creation, and it's better for readability as well. You don't have to go to the top and check what is the  VERSION means here.

Reply:
ok

Reply:
changes done

Reply:
Thanks!